data/screens/encode.kv

```diff
@@ -1,9 +1,9 @@
-#:import random random.random
+#:kivy 1.8.0
```

Comment:
Is it possible to define a minimum resolution?

Reply:
> Is it possible to define a minimum resolution?

I can handle that.